mp/src/game/shared/neo/neo_gamerules.cpp

```diff
@@ -83,7 +83,14 @@ ConVar neo_sv_suicide_prevent_cap_punish("neo_sv_suicide_prevent_cap_punish", "1
 										 "while the other team is holding the ghost, reward the ghost holder team "
 										 "a rank up.",
 										 true, 0.0f, true, 1.0f);
-#endif
+
+ConVar neo_sv_readyup_teamplayersthres("neo_sv_readyup_teamplayersthres", "5", FCVAR_REPLICATED, "The exact total players per team to be in and ready up to start a game.", true, 0.0f, true, (MAX_PLAYERS - 1) / 2);
```

Comment:
Could do with a compile-time assert that 5 is less than (MAX_PLAYERS -1) / 2, otherwise this will cause an error.